core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -320,6 +322,47 @@ public void createContainerCmdHookTest() {
         }
     }
 
+    @Test
+    public void copyToContainerTest() throws Exception {
+
+        try (
+            GenericContainer alpineCopyToContainer = new GenericContainer("alpine:3.2")
+                    .withCommand("sleep 9999")
+        ){
+            alpineCopyToContainer.start();
+
+            final MountableFile mountableFile = MountableFile.forClasspathResource("test_copy_to_container.txt");
+            alpineCopyToContainer.copyFileToContainer(mountableFile, "/home/");
+
+            try (final InputStream response = alpineCopyToContainer
```

Comment:
Since it was requested a few times already, maybe you can add `copyFileFromContainer()` as well? :) We should also check the file content here